addons/nuxeo-platform-spreadsheet/nuxeo-platform-spreadsheet-web/src/main/js/app/ui/editors/document.js

```diff
@@ -42,7 +42,7 @@ class DocumentEditor extends Select2Editor {
   }
 
   getEntryId(item) {
-    return item.uid || item.id;
+    return item && item.uid || item && item.id;
```

Comment:
Kindly optimize it as below:

(item && (item.uid || item.id))

Reply:
@rajatgoel-gl thank you for the comment ,i tried with above code but platform build section in the PR is failing because of that.

Reply:
https://jenkins.platform.dev.nuxeo.com/blue/organizations/jenkins/nuxeo%2F10.10%2Fnuxeo/detail/PR-5225/11/pipeline/56
(Build Platform) Section